saleor/payment/gateways/dummy/__init__.py

```diff
@@ -105,7 +106,13 @@ def process_payment(payment_information: Dict, connection_params) -> Dict:
     responses = [authorize(payment_information, connection_params)]
     if charge_status == ChargeStatus.NOT_CHARGED:
         return responses
+
+    # Only capture half of the authorized amount when set to partially charged
+    if charge_status == ChargeStatus.PARTIALLY_CHARGED:
```

Comment:
I'm not sure about this - we shouldn't know exactly what amount to capture, as partial captures should be only allowed through the dashboard where we should render an input to provide amount.

Reply:
I just would like to make it easier to test since it is dummy gateway anyway. I assume decimal / 2 always works as partial charged.

But yeah maybe it is better to do it in dashboard.